frame/support/src/inherent.rs

```diff
@@ -272,4 +279,14 @@ mod tests {
 
```

Comment:
actually you should also add the Test2 in this above block so that the check extrinsic fail because of SomethingElse.